src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -7,15 +7,46 @@
     private static TalonSRX actuator;
     private static TalonSRX extender;
     private static TalonSRX winch;
+    
+    private static DigitalInput upLimit;
+    private static DigitalInput downLimit;
+    
+    private static boolean actuated;
 
     public Climber(){
         actuator = new TalonSRX(7);
         extender = new TalonSRX(8);
         winch = new TalonSRX(9);
+        
+        upLimit = new DigitalInput(0);
+        downLimit = new DigitalInput(1);
+        
+        actuated = false;
     }
-    public static void actuation(){
-        actuator.set(ControlMode.PercentOutput, 0.3);
+    
+    //Actuates and holds when limit switch is hit
+    public static void actuation() {
+        if(!getUpLimit()) {
+            actuator.set(ControlMode.PercentOutput, 0.3); //Raising
+        }
+        else {
+            hold();
```

Comment:
TODO the hold value needs to be tested to make sure we are not drifting upwards, but that seems like a decent guess to me

Reply:
Yeah that was just a placeholder.